inspektor/check.py

```diff
@@ -16,11 +16,11 @@
 import os
 import tempfile
 
-from . import inspector
```

Comment:
I'm not native so I don't really understand the point of this change. Inspector is the guy watching and making sure everything is as it should be, which IMO fits the PathInspector role. Path checker is in my view similar, but it just checks the path, inspector inspects it deeply and with some authority so I'd keep it as is. But again, I'm not native so I can't really tell.